examples/seismic/model.py

```diff
@@ -304,7 +307,11 @@ def _initialize_physics(self, vp, space_order, **kwargs):
             vs = kwargs.pop('vs')
             self.lam = self._gen_phys_param((vp**2 - 2. * vs**2)/b, 'lam', space_order,
                                             is_param=True)
-            self.mu = self._gen_phys_param(vs**2 / b, 'mu', space_order, is_param=True)
+            # Need to add small value to avoid division by zero
+            if isinstance(vs, np.ndarray):
```

Comment:
Wouldn't this still be the case if `vs` was a float? Where does the div by zero come from?

Reply:
user input setting vs=0 ?

Reply:
>Where does the div by zero come from?

Harmonic mean for half noee evaluation. `1/(.5/mu[x] + .5/mu[x+1]))`

If `vs` is just a float there is no averaging